Resources/Prototypes/_Mono/Entities/Structures/Machines/datafarms.yml

```diff
@@ -0,0 +1,173 @@
+# SPDX-FileCopyrightText: 2025 Onezero0
+#
+# SPDX-License-Identifier: MPL-2.0
+
+
+- type: entity
+  id: BaseDataFarmIndestructible
+  parent: BaseMachineIndestructible
+  name: base datafarm
+  abstract: true
+  components:
+  - type: Sprite
+    sprite: _Mono/Structures/Machines/neapolitan_server.rsi
+    layers:
+    - state: icon
+    - state: unlit
+      shader: unshaded
+      map: [ "enum.PowerDeviceVisualLayers.Powered" ]
+    - state: stripe
+      color: "#b3aa79"
+    - state: stripe2
+      color: "#5b0093"
+  - type: Appearance
+  - type: GenericVisualizer
+    visuals:
+      enum.PowerDeviceVisuals.Powered:
+        enum.PowerDeviceVisualLayers.Powered:
+          True: {visible: true}
+          False: {visible: false}
+  - type: ApcPowerReceiver
+    powerLoad: 40000
+  - type: ContainerContainer
```

Comment:
change to what test tells you to change to
actually why do you even need ContainerContainer here, it should be in a parent machine prototype, i'd say to just modify wizden's BaseMachine and make it into BaseMachineIndestructible and make a new BaseMachine that parents off BaseMachineIndestructible but just adds destructibility